hibernate-core/src/test/java/org/hibernate/orm/test/sql/SqmToSqlTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.orm.test.sql;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.JiraKey;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Yanming Zhou
+ */
+@DomainModel
+@SessionFactory
+public class SqmToSqlTest {
```

Comment:
Not sure about the name of this test, maybe just create a new test method in `org.hibernate.orm.test.query.hql.MultiValuedParameterTest` please.

Reply:
Updated.